vernac/classes.ml

```diff
@@ -276,7 +275,7 @@ let existing_instance glob g info =
   let ctx, r = Term.decompose_prod_decls instance in
     match class_of_constr (Environ.push_rel_context ctx env) sigma (EConstr.of_constr r) with
       | Some (_, ((tc,u), _)) -> add_instance tc info glob c
-      | None -> user_err ?loc:g.CAst.loc
+      | None -> user_err
```

Comment:
Why drop the loc?

Reply:
Because it is not needed (and no longer available in the globref).

Reply:
Locs are never needed but it's still nice to be precise.

>no longer available in the globref

so add a ?loc argument